src/index.html

```diff
@@ -1,16 +1,61 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+  >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page__body">
+    <form
+      data-qa="big"
+      class="search-bar search-bar--big"
+      action="#"
+      method="post"
+    >
+      <a href="#" class="search-button search-button--big">
+        <img
+          src="images/Search.svg"
+          alt="Search icon"
+          class="search-icon search-icon--big"
+        >
+      </a>
+
+      <label class="search-label">
+        <input
+          class="search-input search-input--big"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+
+    <form
+      data-qa="small"
+      class="search-bar search-bar--small"
+      action="#"
+      method="post"
+    >
+      <a href="#" class="search-button search-button--small">
+        <img
+          src="images/Search.svg"
+          alt="Search icon"
+          class="search-icon search-icon--small"
+        >
+      </a>
+
+      <label class="search-label">
+        <input
+          class="search-input search-input--small"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
placeholder from code is not equal the one from demo link, please fix it

Reply:
It is equal